activities.json

```diff
@@ -1069,5 +1069,16 @@
     "org": "WHATWG",
     "title": "timezonechange event",
     "url": "https://github.com/whatwg/html/pull/3047"
+  },
+  {
+    "ciuName": "webgpu",
+    "description": "This specification describes support for accessing modern 3D graphics and GPU computation capabilities on the Web.",
```

Comment:
There's no `id` in this entry, and it needs one.  (Not sure what you used to make it, but the current `activities.py` shouldn't have done that.)  Probably `webgpu` or `gpuweb`.

Reply:
I wrote it by hand. Let me know if I should start over using `activities.py` instead. The "id" is now added.

Reply:
You don't need to start over -- but please check if `./activities.py validate` shows any errors?

Reply:
Will do! Any reason to not run this on CI for every commit?

Reply:
It does run on commits, but I don't think we set it up to run on PRs.  Maybe I should figure out how to do that...